tools/Shared/Microsoft.TemplateEngine.CommandUtils/TestCommand.cs

```diff
@@ -0,0 +1,150 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace Microsoft.TemplateEngine.CommandUtils
+{
+    internal abstract class TestCommand
+    {
+        private readonly LoggerWrapper _loggerWrapper;
+
+        protected TestCommand(ITestOutputHelper log)
```

Comment:
I have a bit mixed feelings about this.
It's nice to have the overload for XUnit logger, on the other hand the reference to XUnit is then forced to all callers.

Maybe we can offer overload accepting custom interface (extracted from LoggerWrapper) and callers would be able to pass that, plus there can be an extra extension class (not required to be imported) conveniently converting from XUnit logger to the interface.

It's not a strong opinion on my side thought.

Reply:
The current use cases are `XUnit` and `ILogger`. If we expose it, l agree, but for now, for simplicity I would prefer to have it as.

Reply:
Thinking about it - we anyway already have a transitive dependency on `XUnit.Abstractions` due to inability to dirrectly call `Verify` (design decision - all functionality is internal and exposed via `internalsvisibleto` only to protocol specific libraries - so we chose to reference `Verify.XUnit`).

tl;dr;: it's good as it is now.